src/pages/Home/components/SectionContent.scss

```diff
@@ -0,0 +1,49 @@
+@import '~theme/base/variables';
+
+.header {
+    margin-bottom: 1.5rem;
+    position: relative;
+
+    &:after {
+        content: '';
+        position: absolute;
+        bottom: -5px;
+        width: 200px;
+        height: 1px;
+        background: $color-grey-light;
+    }
+}
+
+.title {
+    margin: 0;
+    font-size: 2rem;
+    color: $color-grey-dark;
+    $font-family: $font-alternative;
+    text-transform: uppercase;
+}
+
+.description {
+    line-height: 1.75;
+    font-size: 14px;
```

Comment:
Use the `$font-x` variables in #72 when available.

Reply:
Yes! That why I asked for a merge as soon as possible, so we could be using the variables :smile: